packages/core/src/components/button/test/basic/index.html

```diff
@@ -12,6 +12,6 @@
   </head>
 
   <body>
-    <tds-button id="basicButton" text="Button"></tds-button>
+    <tds-button id="basicButton" text="Button" data-testid="tds-button-testid"></tds-button>
```

Comment:
Only thing you can get rid off is "id" in all index.html files accross button test. I think it is now used any more so it is not needed...